pkg/operator/controller/cruft.go

```diff
@@ -186,6 +188,132 @@ func reconcileServiceAccounts(args *callbacks.ReconcileCallbackArgs) error {
 	return nil
 }
 
+func getSpecVersion(version string, crd *extv1.CustomResourceDefinition) *extv1.CustomResourceDefinitionVersion {
```

Comment:
I think is makes sense to move all these changes to callbacks.go as this is not "cruft" that will eventually get deleted.

Reply:
Done!